.gitignore

```diff
@@ -1 +1,2 @@
 env
+.history/
```

Comment:
This looks it is not related with the project

Reply:
Done